docs/docs-content/troubleshooting/cluster-deployment.md

```diff
@@ -94,6 +95,38 @@ Common reasons for why a service may fail are:
 
 6. Check stdout for errors. You can also open a support ticket. Visit our [support page](http://support.spectrocloud.io/).
 
+## Deployment Violates Pod Security
+Cluster deployment fails with the following message. 
+
+```
+Error creating: pods <name of pod> is forbidden: violates PodSecurity "baseline:v<k8s version>": non-default capabilities …
+```
+
+This can happen when the cluster profile uses Kubernetes 1.25 or later and also includes packs that create Pods requiring elevated privileges . 
+
+### Debug Steps
+
+You can change the Pod Security Standards of the namepace where the Pod is being created to address this issue. 
+
+1. Log in to [Palette](https://console.spectrocloud.com).
+
+2. Navigate to the left **Main Menu** and click on **Profiles**. 
+
+3. Select the profile you are trying to deploy the cluster with and choose the layer that represents your pack. 
+
+4. In the YAML file for that pack, under the `pack` field, add a subfield `namespaceLabels` if it doesn't already exist.
+
+5. In the `namespaceLabels` field, add a subfield with the name of your namespace as the key and add `pod-security.kubernetes.io/enforce=privileged,pod-security.kubernetes.io/enforce-version=v<k8s_version>` as its value. Replace `<k8s_version>` with the version of Kubernetes that runs on your cluster. 
+   - If a key matching your namespace already exists here, add the labels to the value corresponding to that key.
+   - For example, if you the pack creates a namespace called `monitoring`, add the labels to the `monitoring` namespace:
```

Comment:
Missing word. Should it be "For example, if your pack creates a namespace called `monitoring`, add the labels to the `monitoring` namespace as shown in the example below.